specs/phase0/p2p-interface.md

```diff
@@ -626,6 +625,8 @@ The `ErrorMessage` schema is:
 *Note*: By convention, the `error_message` is a sequence of bytes that MAY be interpreted as a UTF-8 string (for debugging purposes).
 Clients MUST treat as valid any byte sequences.
 
+The responder MAY penalise peers that concurrently open more than two streams for the same request type, for the protocol IDs defined in this specification.
```

Comment:
I think this should use `MAX_CONCURRENT_REQUESTS`?

Reply:
oops, I had that in my local changes, thanks!